src/tab/TabGroup.ts

```diff
@@ -150,6 +150,8 @@ export class TabGroup extends OmniElement {
                     display: flex;
                     flex-direction: row;
                     align-items: center;
+                    overflow-x: auto;
+                    overflow-y: hidden;
```

Comment:
Please make a CSS var for this.

Reply:
Added accordingly with update to JS doc comment.